q2_feature_classifier/_cutter.py

```diff
@@ -168,6 +168,10 @@ def extract_reads(sequences: DNASequencesDirectoryFormat, f_primer: str,
     q2_types.DNAFASTAFormat
         containing the reads
     """
+    if min_length > trunc_len and trunc_len > 0:
+        raise ValueError('Your minimum length is larger than your truncation '
```

Comment:
"larger than" --> "greater than or equal to"

Reply:
I misinterpreted this initially, and changed the check to `min_len >= trunc_len` instead of changing the comment, but now that I think about it, will it be a problem if `min_len == trunc_len`? Shouldn't it only be an issue if the minimum length is greater?

Reply:
Also, just to be clear, on that last commit the check has been changed to `min_len >= trunc_len` and the message is unchanged in accordance with my initial misinterpretation of your comment. Easy enough to fix though obviously.

Reply:
good call @Oddant1 I think I was misinterpreting this before.

But the plot thickens... it occurs to me that this validation should also include trim_left, since that will impact the length. Should this instead be `if min_length > trunc_len - trim_left`? And also change the error message accordingly.